Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/border.h

```diff
@@ -152,9 +152,9 @@ namespace Polygon_mesh_processing {
   * but `face(h, pmesh)` does not belong to the patch.
   *
   * @tparam PolygonMesh model of `HalfedgeGraph`. If `PolygonMesh`
-  *  has an internal property map
+  *  has an internal not writable property map
```

Comment:
`read-only` maybe ?

Reply:
The wording I see the most is "non modifiable", but in the documentation of Boost property map, the wording is "readable", "writable":
https://www.boost.org/doc/libs/1_71_0/libs/property_map/doc/property_map.html

Reply:
Amnd non instead of not

Reply:
for lvalue maps, mutable is also used in boost.